### ISSUE TITLE
Request to add 55Zh6-U (recent upgrade of Tall Rack UYE)

### ISSUE THREAD

**Issue by GIGANTO-pithecus** [PygmalionOfCyprus/cmo-db-requests#3655]
### DB Selector

DB3K

### DBID to Copy

Facility 1616 , sensor 2522

### Hypothetical

No

### Name

55Zh6-U Nebo-U "Tall Rack"

### Country

Russia

### Service

Air Force

### In Commission

2017-present

### Number in Service

5-10, limited

### Sensors

-increase max range to 700km (378nm)

### Other Changes

recent (2017-present) upgrade of older 1990's era Tall Rack UYe with a newer transmitter, solid state components, new software and updated hardware. Same antenna height, same VHF band

Improved range max: 700km
Frequency bands: VHF (A/B) similar to 1L31-SV, Nebo-SVU
Radar properties: PESA, TWS, Pulse Doppler (LDSL), IFF

specs of upgraded 55zh6-1 below:
![nebo-u 1](https://github.com/PygmalionOfCyprus/cmo-db-requests/assets/134186902/23c4a756-fe37-4d73-acc7-9ec0be64f988)

range/azimuth error:
![nebo-u 2](https://github.com/PygmalionOfCyprus/cmo-db-requests/assets/134186902/56b5ae0e-37ef-4fff-9169-7e30fdd3ba72)

in limited service, first delivery made 2017. 

### Sources

https://www.radartutorial.eu/19.kartei/02.surv/karte039.en.html
https://en.topwar.ru/93543-v-zvo-postupila-sovremennaya-rls-nebo-u.html
https://twitter.com/russian_defence/status/881859899965153280?lang=da

**Comment by HVAA** [comment]
Bit more common than you might think.  From GE two here 44.679002870983496, 33.5543401752197. One here 44.529923228257495, 33.459212432743854 (a couple of hundred metres from the Big Bird) and another one here 45.119418197224725, 33.990170307893564 - all in Crimea

**Comment by GIGANTO-pithecus** [comment]
> Bit more common than you might think. From GE two here 44.679002870983496, 33.5543401752197. One here 44.529923228257495, 33.459212432743854 (a couple of hundred metres from the Big Bird) and another one here 45.119418197224725, 33.990170307893564 - all in Crimea

Yes, these are most likely the upgraded Nebo-U Tall Rack systems. Most, if not all, are undergoing modifications/software/hardware upgrades to the 55zh6-U standard. good eye

**Comment by HVAA** [comment]
Also A band (NATO) for deconfliction reasons. Radars only operate in certain bands otherwise they give an easy 'kill' when they are jammed by comms bands (for effectively no planned effort).  So B band (for the main part is ruled out. Radars will sit between VHF (140 MHZ) to comm UHF (225 MHZ). 
These are pulse only radars, not PD.  At this frequency there is no need for PD.  Blind speeds are naturally high, main lobe will hug the ground and delay line cancellation will effectively provide an MTI function.
